third-culture-software/bhima
# Fix form warning messages in shipment creation form

Fixes the warning messages in the Shipment creation form to be more helpful.

Made the estimated delivery date required.

Removed the "Exit type set" info message since there is no need for that in the Shipment form since it is always a transfer to a depot.  It was a bit confusing before.

Closes https://github.com/IMA-WorldHealth/bhima/issues/6490

**NOTE:** One thing that was not addressed in this PR is whether selecting a condition for each item is required before submitting.   If it is not set, the form can be submitted, but the condition for the item is null for the resulting shipment.   We should think about this.   And it is not clear that the condition is needed as part of the shipment itself since that could be tracked with asset scans.

TESTING
- bhima_test
- Shipments > Create Shipment
  - Try clicking on the [Submit] button immediately
  - Try different orders of creating the items and filling out the form and make sure the warnings make sense.
- Then try Stock Exit (which usese the same StockExit
